core/src/main/java/com/linecorp/armeria/common/SerializationFormat.java

```diff
@@ -67,10 +75,16 @@
                                    "unknown", create("application", "x-unknown")));
 
         // Load all serialization formats from the providers.
-        ServiceLoader.load(SerializationFormatProvider.class,
-                           SerializationFormatProvider.class.getClassLoader())
-                     .forEach(p -> p.entries().forEach(e -> register(mutableUriTextToFormats,
+        final List<SerializationFormatProvider> providers = Streams.stream(
+                ServiceLoader.load(SerializationFormatProvider.class,
+                                   SerializationFormatProvider.class.getClassLoader())).collect(
+                toImmutableList());
+        if (!providers.isEmpty()) {
+            logger.info("Found {}: {}", SerializationFormatProvider.class.getSimpleName(), providers);
```

Comment:
How about using `Load` instead of `Found` to meet the term with `ServiceLoader.load`?

Reply:
Fixed. 😉